sdk/spring/spring-cloud-azure-appconfiguration-config-web/src/main/java/com/azure/spring/cloud/config/web/package-info.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 /**
- * Package contains classes for automatic refresh.
+ * Package contains classes for enabling auto refresh of Azure App Configuration stores
  */
 package com.azure.spring.cloud.config.web;
```

Comment:
No need for this class, since there's no public classes.

Reply:
The build seems to fail if this file is removed.

Reply:
Let me try locally.